### PR TITLE
Create configuration view

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/view.dart [hunk 1]
```diff
@@ -192,3 +179,135 @@ class AvailablePackagesViewPartial {
...
+      ..classes.add('configure-package');
+    _tagsContainer = new DivElement()
+      ..classes.add('configure-package__tags');
+    tagList = new ConfigurationViewTagListPartial();
+    tagResponses = new ConfigurationViewTagResponsesPartial();
+
+    _tagsContainer.append(tagList.tagListElement);
+    _tagsContainer.append(tagResponses.tagResponsesElement);
+    configurationViewElement.append(HeadingElement.h1()
+      ..classes.add('configure-package-title')
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: nit: 
```suggestion
      ..classes.add('configure-package__title')
```

File: webapp/lib/view.dart [hunk 2]
```diff
@@ -192,3 +179,135 @@ class AvailablePackagesViewPartial {
...
+      for (var response in responseSet) {
+        body.append(new ParagraphElement()
+        ..classes.add('configure-package__tag-responses-body-item')
+        ..attributes.addAll({'contenteditable': 'true', 'parent-tag': tag, 'language': '$language' ,'index': '$responseIndex'})
+        ..text = response
+        ..onBlur.listen((event) {
+          var reponseElement = (event.currentTarget as Element);
+          var parentTag = reponseElement.attributes['parent-tag'];
+          var editedResponse = {'index': reponseElement.attributes['index'], 'language': reponseElement.attributes['language'] ,'text': reponseElement.text};
+          controller.command(controller.UIAction.editConfigurationTagResponse, new controller.ConfigurationResponseData(parentTag: parentTag, editedResponse: editedResponse));
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: would it make sense to make `index`, `language` and `text` properties of `controller.ConfigurationResponseData`, alongside `parentTag`?

File: webapp/lib/view.dart [hunk 3]
```diff
@@ -192,3 +179,135 @@ class AvailablePackagesViewPartial {
...
+
+  void renderResponses(String tag, Map<String, List<String>> responses) {
+    tagResponsesElement.children.clear();
+    _tagResponsesHeader.children.clear();
+    _tagResponsesBody.children.clear();
+    responses.forEach((language, responseSet) {
+      _tagResponsesHeader.append(new HeadingElement.h5()..text = language);
+      var body = new DivElement()..classes.add('configure-package__tag-responses-body-items');
+      int responseIndex = 0;
+      for (var response in responseSet) {
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: if you need to iterate through a list and keep track of an index, prefer to use a `for i` loop, e.g.
```suggestion
      for (int i = 0; i < responseSet.length; i++) {
        var response = responseSet[i];
```

File: webapp/lib/view.dart [hunk 4]
```diff
@@ -192,3 +179,135 @@ class AvailablePackagesViewPartial {
...
+      ..classes.add('configure-package__tag-responses-body');
+  }
+
+  void renderResponses(String tag, Map<String, List<String>> responses) {
+    tagResponsesElement.children.clear();
+    _tagResponsesHeader.children.clear();
+    _tagResponsesBody.children.clear();
+    responses.forEach((language, responseSet) {
+      _tagResponsesHeader.append(new HeadingElement.h5()..text = language);
+      var body = new DivElement()..classes.add('configure-package__tag-responses-body-items');
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: prefer not to use "body" as a variable name it tends to mainly be used to refer to the <body> element. how about `items`? (and `configure-package__tag-responses-items`)

File: webapp/web/configuration.css [hunk 5]
```diff
@@ -0,0 +1,149 @@
+.configure-package {
+    display: flex;
+    flex-direction: column;
+}
+
+.configure-package-title {
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: nit: 
```suggestion
.configure-package__title {
```

File: webapp/lib/view.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/view.dart]: yes...I think that's a better way. Changed

File: webapp/lib/view.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Thanks for catching this.

File: webapp/lib/view.dart (hunk 3 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Yes. This is the right one.

File: webapp/lib/view.dart (hunk 4 above)
Comment (ktkization) [reply webapp/lib/view.dart]: I agree. Changed it.

File: webapp/web/configuration.css (hunk 5 above)
Comment (ktkization) [reply webapp/web/configuration.css]: Thanks for catching this.

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization !

**Pull request merged**